spring-integration-cassandra/src/main/java/org/springframework/integration/cassandra/outbound/CassandraMessageHandler.java

```diff
@@ -69,13 +70,23 @@
 	private MessageProcessor<Statement> statementProcessor;
 
 	private EvaluationContext evaluationContext;
+	
+	private String query;
+	
 
 	public CassandraMessageHandler(CassandraOperations cassandraTemplate) {
-		this(cassandraTemplate, Type.INSERT);
+		this(cassandraTemplate, OperationType.INSERT);
 	}
+	
 
-	public CassandraMessageHandler(CassandraOperations cassandraTemplate, CassandraMessageHandler.Type queryType) {
-		Assert.notNull(cassandraTemplate, "'cassandraTemplate' must not be null.");
+	public void setQueryType(OperationType queryType) {
+		this.queryType = queryType;
+	}
+
+	public CassandraMessageHandler(CassandraOperations cassandraTemplate,
+			CassandraMessageHandler.OperationType queryType) {
+		Assert.notNull(cassandraTemplate,
+				"'cassandraTemplate' must not be null.");
```

Comment:
To be honest, I'm not sure that you should rewrite the code format so much.
I can guess that you may have 90 line length config, but we with Gary prefer 120.
Sorry, if that was confusing for you.